packages/components/src/FormField/FormFieldTypes.ts

```diff
@@ -113,6 +114,15 @@ export interface CommonFormFieldProps {
    * Set the component to the given value.
    */
   readonly value?: string | number | Date;
+
+  /**
+   * Add a clear action on the input that clears the value.
+   *
+   * You should always use `while-editing` if you want the input to be
+   * clearable. if the input value isn't editable (i.e. `InputDateTime`) you can
```

Comment:
This comment is referencing a mobile-only component, but I guess the example is still valid?

Reply:
Good point, I can change it to `InputTime`. Even though we can edit it, the timer picker is probably more used than typing on it